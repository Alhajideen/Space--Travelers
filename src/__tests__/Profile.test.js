import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Rocket from '../components/rockets/Rockets';
import configureStore from '../redux/store';

it('renders correctly', () => {
  const rocket = renderer
    .create(<Provider store={configureStore}><Rocket /></Provider>)
    .toJSON();
  expect(rocket).toMatchSnapshot();
});
