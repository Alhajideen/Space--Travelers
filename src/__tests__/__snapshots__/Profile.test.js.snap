// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="rockets-container"
>
  <ul
    className="rockets-list"
    data-testid="rocket-list"
  />
</section>
`;
